### DISCUSSION TITLE
Zen Based Auto theme

### DISCUSSION THREAD

**Discussion by RABINGG** [zen-browser/desktop#1582] [Ideas]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [ ] I am not filing an enhancement request.

### What happened?

Kudos to the Zen theme. Zen based theme suits the browser well rather than the default light, dark and system - auto theme. It would be nice to have a zen based auto theme (Zen dream for light, Zen Galaxy for dark).

Or else provide option to select light theme and dark theme separately. And option to adopt system theme or custom schedule option.

### Reproducible?

- [ ] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.2 (64-bit)

### What platform are you seeing the problem on?

Windows

**Comment by linear** [comment]
<p><a href="https://linear.app/zen-browser/issue/ZEN-1642/zen-based-auto-theme">ZEN-1642 Zen Based Auto theme</a></p>

**Comment by Trikzon** [comment]
I've used the extension [automaticDark](https://addons.mozilla.org/en-US/firefox/addon/automatic-dark/) with Firefox in the past and it looks like it works with Zen as well

**Reply by RABINGG** [reply]
Thanks will try it out. But a auto theme is more convenient
